docs/docusaurus.config.js

```diff
@@ -35,6 +35,17 @@ const config = {
                 docs: {
                     routeBasePath: "/",
                     sidebarPath: require.resolve('./sidebars.js'),
+                    lastVersion: '2.5',
+                    includeCurrentVersion: false,
+                    versions: {
+                        '2.5': {
+                            banner: 'none'
+                        },
+                        '2.6': {
+                            label: '2.6 (development)',
```

Comment:
Now that we released 2.6, this portion needs a bit of tweaking I guess. Let's chat on whether it makes sense to just start a 2.7 version and bump all these numbers by 1, or just swap which version we call current and allow a banner on 2.5

Reply:
I just resolved this for this commit. 
For future reference, our decision was:
No banners for maintained versions, `unmaintained` banner for unsupported versions, `unreleased` banner for development versions